File: templates/Uwp/_composition/_shared/Page.Ink.AddBasicInkServicesVB/EventHandlers/Ink/RemoveEventArgs.vb
Imports Windows.UI.Input.Inking

Namespace EventHandlers.Ink
    Public Class RemoveEventArgs
        Public Property RemovedStroke As InkStroke

        Public Sub New(removedStroke As InkStroke)
            Me.RemovedStroke = removedStroke
        End Sub
    End Class
End Namespace

File: templates/Uwp/_composition/_shared/Page.Ink.AddBasicInkServices/EventHandlers/Ink/RemoveEventArgs.cs
using Windows.UI.Input.Inking;

namespace Param_ItemNamespace.EventHandlers.Ink
{
    public class RemoveEventArgs
    {
        public InkStroke RemovedStroke { get; set; }

        public RemoveEventArgs(InkStroke removedStroke)
        {
            RemovedStroke = removedStroke;
        }
    }
}
